app/src/pages/settings/ResetPasswordDialog.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { graphql, useMutation } from "react-relay";
+
+import {
+  Button,
+  Dialog,
+  Flex,
+  Form,
+  TextField,
+  View,
+} from "@arizeai/components";
+
+import { useNotifyError, useNotifySuccess } from "@phoenix/contexts";
+
+import { ResetPasswordDialogMutation } from "./__generated__/ResetPasswordDialogMutation.graphql";
+
+const MIN_PASSWORD_LENGTH = 4;
+
+export type ResetPasswordFormParams = {
+  newPassword: string;
+  confirmPassword: string;
+};
+
+export function ResetPasswordDialog({
+  userId,
+  onClose,
+}: {
+  userId: string;
+  onClose: () => void;
+}) {
+  const notifyError = useNotifyError();
+  const notifySuccess = useNotifySuccess();
+  const [commit, isCommitting] = useMutation<ResetPasswordDialogMutation>(
+    graphql`
+      mutation ResetPasswordDialogMutation($input: PatchUserInput!) {
+        patchUser(input: $input) {
+          __typename
+        }
+      }
+    `
```

Comment:
not sure if this fails server-side but we need to block this flow if the user doesn't use local auth

Reply:
i believe it does